packages/frontend-api/src/Model/Resolver/Products/ProductResolver.php

```diff
@@ -13,6 +13,9 @@
 use Shopsys\FrameworkBundle\Model\Product\Product;
 use Shopsys\FrameworkBundle\Model\Product\ProductFacade;
 
+/**
+ * @deprecated Use ProductByUuidResolver instead
```

Comment:
I am missing reason why is this class renamed. It is just obout naming or it has some other reason I do not see now?

Reply:
The main reason is that I need different dependencies – keeping the current resolver would mean I need to deprecate current dependencies and add the new one in the BC manner. 
New resolver allows me to have dependencies clear, I can deprecate tho whole class, which is more understandable and I can switch to the previous resolver really easy if necessary.